src/foam/i18n/Locale.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.CLASS({
+  package: 'foam.i18n',
+  name: 'Locale',
+
+  properties: [
+    {
+      class: 'String',
+      name: 'locale',
```

Comment:
can we not make Enums for these three?  They are never going to change - or very very infrequently.  Simplify human setup - can't make a mistake.